services/outputhost/cgcache.go

```diff
@@ -486,11 +489,10 @@ func (cgCache *consumerGroupCache) refreshCgCache(ctx thrift.Context) error {
 
 		// Now we have the cgCache. check and load extent for all extents
 		for _, cge := range cgRes.GetExtents() {
-			nExtents++
-			errR := cgCache.loadExtentCache(dstDesc.GetType(), cge)
-			if errR != nil {
-				return errR
+			if cgCache.checkSingleCGVisible(ctx, cge) {
+				nSingleCGExtents++
 			}
+			cgCache.loadExtentCache(dstDesc.GetType(), cge)
```

Comment:
you are now ignoring the error from loadExtentCache (previously not). If loadExtentCache() is never supposed to fail, change the function signature to not return an error.